devops/arangodb.yml

```diff
@@ -42,11 +47,39 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
+    - name: Copy the compressed ssl certificate files to the server
+      copy:
+        src: '{{ ssl_cert_path }}'
+        dest: /home/ubuntu/
+        follow: yes
+        mode: u=rw,g=,o=
+
+    - name: Combine the chain and the private key into one file (required by arango)
+      shell:
+        chdir: /home/ubuntu/
+        cmd: cat fullchain.pem privkey.pem > server.pem
+
```

Comment:
The command here _is_ idempotent, which is good, but in general when using the `shell` directive, you should put in logic to ensure this task doesn't run if it has already run before. In this case, you can add the property that says this task creates a file (maybe it's `creates`; I can't remember off the top of my head).

Reply:
Added creates in c3f8ef2

Reply:
Do you think it makes sense to have it use creates? What if we reprovision on a server to update the keyfile? Then the server.pem file wouldn't be updated. I guess the question is, do you think we'll use this file to update the certificate?

Reply:
Ideally we'd want to make the task depend on those two files as dependencies somehow. Probably not worth figuring this out right now for the reasons you mentioned in the other thread. In this case, like I said originally, you don't have to have the `creates` (since the concatentation is idempotent).

Reply:
I removed the `creates` because it will complicate things when we run this playbook multiple times on the same provider